msteams-platform/bots/how-to/conversations/command-bot-in-teams.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Notification bot in Teams
+author: surbhigupta
+description: Learn how a command bot works in Teams, and to add command and responses.
+ms.topic: conceptual
+ms.author: v-amprasad
+ms.localizationpriority: high
+---
+
+# Command bot in Teams
+
+Microsoft Teams allows you to automate simple and repetitive tasks in a conversation. You can build a command bot that can respond to simple commands sent in chats with Adaptive Cards. You can create a command bot template in Teams that responds to chat commands by displaying UI using an Adaptive Card. This enables users to send messages in Teams and your app can provide a response as required.
+
+The command bot template is built using the TeamsFx SDK, which provides a simple set of functions over the Microsoft Bot Framework to implement the scenario. Command bot can be used in different scenarios such as checking ticket status, and retrieve help information.
+
+:::image type="content" source="../../../assets/images/command-bot-teams/commandbot-flowchart1.png" alt-text="creating command bot app with adaptive card flow chart" lightbox="../../../assets/images/command-bot-teams/commandbot-flowchart1.png":::
+
```

Comment:
Actually I feel like this chart isn't quite helpful, it's hard to see the pictures inside the flows, maybe we should get rid of that?

Reply:
Yes, the font is too small, I am updating it please confirm if the updated one looks better?